src/gui/DatabaseTabWidget.cpp

```diff
@@ -764,6 +767,11 @@ void DatabaseTabWidget::emitActivateDatabaseChanged()
     Q_EMIT activateDatabaseChanged(currentDatabaseWidget());
 }
 
+void DatabaseTabWidget::emitDatabaseUnlockedFromDbWidgetSender()
```

Comment:
The name of the function is weird because its intended to be a SLOT. Might be able to consolidate some code and clean this up a little.